clients/src/test/java/org/apache/kafka/common/metrics/stats/MeterTest.java

```diff
@@ -0,0 +1,78 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.kafka.common.metrics.stats;
+
+import static org.junit.Assert.assertEquals;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+
+import org.apache.kafka.common.MetricName;
+import org.apache.kafka.common.metrics.CompoundStat.NamedMeasurable;
+import org.apache.kafka.common.metrics.MetricConfig;
+import org.junit.Test;
+
+public class MeterTest {
+
+    private static final double EPS = 0.0000001d;
+
+    @Test
+    public void testMeter() {
+        Map<String, String> emptyTags = Collections.emptyMap();
+        MetricName rateMetricName = new MetricName("rate", "test", "", emptyTags);
+        MetricName totalMetricName = new MetricName("total", "test", "", emptyTags);
+        Meter meter = new Meter(rateMetricName, totalMetricName);
+        List<NamedMeasurable> stats = meter.stats();
+        assertEquals(2, stats.size());
+        NamedMeasurable rate;
+        NamedMeasurable total;
+        if (stats.get(0).name().equals(rateMetricName)) {
+            rate = stats.get(0);
+            total = stats.get(1);
+        } else {
+            rate = stats.get(1);
+            total = stats.get(0);
+        }
```

Comment:
Do we need the if/else? Since this is a unit test, it seems OK to just assert that the first element is the rate and the second is the total.